### PR TITLE
Add Brick Hill

### PR DESCRIPTION
**Pull request by dragonismcode**
**Proof of working:**
image
image

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
fix codacy issues

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors

**Comment by dragonismcode** [comment]
I cannot seem to find a fix to these errors. If you can help me, that would be greatly appreciated.

File: websites/B/Brick Hill/presence.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+const presence = new Presence({
+  clientId: "797879854343127040"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/B/Brick Hill/presence.ts:5]: merge consts

```
const a = "b",
  b = "c";
```

File: websites/B/Brick Hill/presence.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+var title: any;
```
Comment (Bas950) [inline websites/B/Brick Hill/presence.ts:7]: Dont use var, dont use any

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**